README.md

```diff
@@ -42,6 +43,7 @@ You can refer to either the [main documentation](https://docs.descope.com/custom
 `<PROJECT_ID>` as part of the URL path is required to use the desired Descope's `PROJECT_ID`  
 `flow` query parameter is optional. If none provided the default flow is `sign-up-or-in`  
 `debug` query parameter is optional. If debug mode is needed use `debug=true`
+`tenant` query parameter is optional. If present, then you will be able to authenticate via SSO, without having to first specify an email with an input screen in your flow.
```

Comment:
same, I suggest to have it after the `flow`

Reply:
It should work with either tenantID/tenantName/emailDomain right?
Did you test it? maybe worth adding it to the README.

Reply:
Changed the README. Tenant Name didn't seem to work but Tenant ID and email domain works, so I added to the README.